packages/accounts/tsconfig.build-cjs.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.build.json",
+  "compilerOptions": {
+    "esModuleInterop": true,
```

Comment:
We need esModuleInterop for Axios to work per here https://www.npmjs.com/package/axios?activeTab=readme#typescript.  It doesn't hurt to be in both build files so I just added it to both